funcx_endpoint/funcx_endpoint/executors/high_throughput/executor.py

```diff
@@ -259,7 +260,7 @@ def __init__(self,
         self.heartbeat_period = heartbeat_period
         self.poll_period = poll_period
         self.suppress_failure = suppress_failure
-        self.run_dir = '.'
```

Comment:
This is redundant because you are setting `executor.run_dir` explicitly in the interchange.

Reply:
I think it has been taken care of at another place: in `funcx_endpoint/executors/high_throughput/executor.py`, it does `logdir="{}/{}".format(self.run_dir, self.label),`, which adds the `label` part. So
- If the user has `run_dir='/tmp'` in the config, the log should appear at `/tmp/HighThroughputExecutor`
- If the user do not have `run_dir` in the config the log should apprear at `~/.funcx/epname/HighThroughputExecutor`

Reply:
Yes, in the updated code, `self.run_dir = run_dir`, where `run_dir=None` as default. If the user does not specify `run_dir` in the config, both `self.run_dir` and `run_dir` would be `None`, which would trigger the assignment (to `~/.funcx/endpoint_name`) in the interchange.